tools/clang/test/CodeGenSPIRV/compute.byte-address-buffer.load.hlsl

```diff
@@ -0,0 +1,33 @@
+// Run: %dxc -T cs_6_0 -E main
+
+// CHECK: %_ptr_Uniform_uint = OpTypePointer Uniform %uint
+// CHECK: %myBuffer = OpVariable %_ptr_Uniform_type_ByteAddressBuffer Uniform
+ByteAddressBuffer myBuffer;
+
+[numthreads(1, 1, 1)]
+void main() {
+  uint addr = 0;
+
+// CHECK: [[addr1:%\d+]] = OpLoad %uint %addr
+// CHECK-NEXT: [[word_addr:%\d+]] = OpShiftRightLogical %uint [[addr1]] %uint_2
+// CHECK-NEXT: [[load_ptr:%\d+]] = OpAccessChain %_ptr_Uniform_uint %myBuffer %uint_0 [[word_addr]]
+// CHECK-NEXT: {{%\d+}} = OpLoad %uint [[load_ptr]]
+  uint word = myBuffer.Load(addr);
+
```

Comment:
I'd suggest to add cases for `Load2()` and `Load3()`. Each of these load method is distinct. Although the way we handle them in the implementation is similar.  (It's not like floatN, where N is a parameter, so we only need a representative for N. Here we have three different methods.)